frontend/components/course/EnrollmentStatus.tsx

```diff
@@ -17,15 +22,17 @@ interface IProps {
 export const EnrollmentStatus: FC<IProps> = ({ course }) => {
   const [isModalVisible, setModalVisible] = useState(false);
   const { t } = useTranslation("course-application");
+  const { keycloak } = useKeycloak<KeycloakInstance>();
 
-  const { data, loading, error } = useAuthedQuery<CourseWithEnrollment>(
-    COURSE_WITH_ENROLLMENT,
-    {
-      variables: {
-        id: course.Id,
-      },
-    }
-  );
+  const { data, loading, error } = useAuthedQuery<
+    CourseWithEnrollment,
+    CourseWithEnrollmentVariables
+  >(COURSE_WITH_ENROLLMENT, {
+    variables: {
+      id: course.Id,
+      authId: keycloak?.subject,
```

Comment:
What do you think of having a hook that directly gives you the `authId`? I tend to find it tedious to work with the keycloak hook.

Reply:
I think we can easily add this in a new PR. So I'm merging this to unblock other things for now.

Reply:
Good point, feel free to drop of a PR ;)